### PR TITLE
Templates Round 2

### PR DESCRIPTION
**Pull request by tpreusse**
Notable Changes
- template creators
- add Meta template
- `FigureImage` props are now flat: `data={{src: '', alt: ''}}` becomes `src={''} alt={''}`
- add `center` size to `Figure` (essay template with figure in front of title, center wrap would be impractical here)

### CODE REVIEW

File: src/components/Figure/Image.js [hunk 1]
```diff
@@ -8,15 +8,13 @@ const styles = {
   })
 }
 
-const Image = ({ data: { src, alt }, attributes = {} }) => {
+const Image = ({ src, alt, attributes = {} }) => {
```
Comment (uxengine) [inline src/components/Figure/Image.js]: Please also update 
TeaserFront/Image.js
TeaserFront/Split.js
back to the original src/alt props.

File: src/components/Figure/docs.md [hunk 2]
```diff
@@ -2,24 +2,26 @@
 
 A `<Figure />` contains an `<FigureImage>` and an optional `<FigureCaption>`.
```
Comment (uxengine) [inline src/components/Figure/docs.md]: ... a `<FigureImage>`

File: src/components/Typography/Editorial.js [hunk 3]
```diff
@@ -158,6 +159,7 @@ export const Emphasis = ({ children, attributes, ...props }) => (
 
 const link = css({
   textDecoration: 'underline',
+  textDecorationSkip: 'ink',
```
Comment (uxengine) [inline src/components/Typography/Editorial.js]: This repeated combo of textDecoration styles could probably go to lib/styleMixins.js

File: src/templates/Editorial/docs.md [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+import createEditorialSchema from '@project-r/styleguide/lib/templates/Editorial'
+
+const schema = createEditorialSchema({
+  titleBlockAppend: <div>Share Actions</div>
+})
+```
+
+`createEditorialSchema` take an optional options object with follow keys:
+
+- `TitelBlockHeadline`, the title react component
```
Comment (uxengine) [inline src/templates/Editorial/docs.md]: For conistency: Title... rather than Titel... 
Here and in
templates/Editorial/index.js
templates/Meta/index.js

**Comment by tpreusse** [comment]
@uxengine thank you for the feedback. Pushed the changes.

**Pull request merged**